keras_nlp/models/albert/albert_backbone.py

```diff
@@ -271,6 +271,10 @@ def get_config(self):
         )
         return config
 
+    @property
```

Comment:
I agree with @NusretOzates, the `self._token_embedding` approach seems cleaner than relying on a specific layer name. We already have several other similar class variables.

Reply:
It doesn't work for models with `TokenAndPositionEmbedding` layer. Keras considers it as a separate embedding layer, and errors out when we try to load preset checkpoints. Hence, this elaborate-ish solution.

Reply:
I know I am dumping too much context, but for those interested in going deeper...

The `__setattr__` tracking is deduped, so for Bert, where the token embedding is already a sublayer of the model directly, there is no issue here. `self.some_properly = direct_layer_of_model` has no issues. But Roberta for example will have the token embedding as a nested layer. `self.some_property = nested_layer_of_model` will change our checkpoint structure! This is what @NusretOzates was mentioning above.

Also thanks @NusretOzates for that writeup! Very helpful!